distribution/docs/src/main/resources/content/_installing/os-services.adoc

```diff
@@ -0,0 +1,28 @@
+:title: Managed Services
+:type: startingIntro
+:status: published
+:summary: Install Processes as Services.
+:project: ${branding}
+:order: 03
+
+=== {title}
+
+The lifecycle of ${branding} and Solr processes can be managed by the operating
+system. The procedures for installing a processes as a service varies by operating
+system and the particular init manager used by the operating system.
+The ${branding} documentation provides instructions to install
+${branding} and as managed services for supported platform. However, the
+documentation cannot account for all possible environments. Please consult the
```

Comment:
:question: Should this say "... and its init manager if the instructions in this document are inadequate."?